### ISSUE TITLE
"Enter your account password as it was when you registered" 

### ISSUE THREAD

**Issue by neekless** [standardnotes/mobile#519]
I have changed my password some time back. Now whenever I install this app, I have to input my original password to unlock my notes. I still remember my old password for now, but the whole point of changing my password is so I won't have to use my old password anymore. Is there any way to prevent this from happening?
![Screenshot_20211228-161300](https://user-images.githubusercontent.com/22426822/147544808-ab2cfda1-d575-43e1-b06a-04f3056316ff.png)

**Comment by moughxyz** [comment]
Hmm this is an error recovery mechanism. Essentially something didn't sync properly so now you have to enter your old password to decrypt this key. But after that it should properly correct itself so that this doesn't keep appearing. Strange that it's not.

**Comment by neekless** [comment]
> Hmm this is an error recovery mechanism. Essentially something didn't sync properly so now you have to enter your old password to decrypt this key. But after that it should properly correct itself so that this doesn't keep appearing. Strange that it's not.

Yeah it happens every time I newly install the app. Is there anything I can do to generate a new key with the new password or something?
